vllm/worker/cpu_worker.py

```diff
@@ -150,6 +150,9 @@ def __init__(
         if self.is_driver_worker:
             assert self.rank == 0, "The driver worker must have rank 0."
 
+        # try to initialize intel openmp optimized tunings
+        init_kmp_env()
```

Comment:
Please double-check whether the env variables are correctly used by the libraries. In my personal experience, some env variables have to be set before the libraries are loaded.